x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/TrainedModelLocation.java

```diff
@@ -7,16 +7,10 @@
 
 package org.elasticsearch.xpack.core.ml.inference.trainedmodel;
 
-import org.elasticsearch.Version;
 import org.elasticsearch.common.io.stream.NamedWriteable;
-import org.elasticsearch.xpack.core.ml.inference.TrainedModelConfig;
 import org.elasticsearch.xpack.core.ml.utils.NamedXContentObject;
 
 public interface TrainedModelLocation extends NamedXContentObject, NamedWriteable {
 
     String getResourceName();
-
-    default Version getMinimalCompatibilityVersion() {
```

Comment:
This method is not actually used anywhere

Reply:
@davidkyle is it ok this is not actually used?

Reply:
Yes please delete it. 

The streaming compatibility check is performed elsewhere, it's not needed here.
Thanks
https://github.com/elastic/elasticsearch/blob/6d009a38c79c2bdea764a4f567671e3e3827700a/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/TrainedModelConfig.java#L250